internal/cli/ops_manager_security_enable.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/convert"
+	"github.com/mongodb/mongocli/internal/description"
+	"github.com/mongodb/mongocli/internal/flags"
+	"github.com/mongodb/mongocli/internal/messages"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type opsManagerSecurityEnableOpts struct {
+	*globalOpts
+	mechanisms []string
+	store      store.AutomationPatcher
+}
+
+func (opts *opsManagerSecurityEnableOpts) init() error {
+	if opts.ProjectID() == "" {
+		return errMissingProjectID
+	}
+	var err error
+	opts.store, err = store.New()
+	return err
+}
+
+func (opts *opsManagerSecurityEnableOpts) Run() error {
+	current, err := opts.store.GetAutomationConfig(opts.ProjectID())
+
+	if err != nil {
+		return err
+	}
+
+	if err = convert.EnableMechanism(current, opts.mechanisms); err != nil {
+		return err
+	}
+	if err = opts.store.UpdateAutomationConfig(opts.ProjectID(), current); err != nil {
+		return err
+	}
+
+	fmt.Print(messages.DeploymentStatus(config.OpsManagerURL(), opts.ProjectID()))
+
+	return nil
+}
+
+// mongocli ops-manager security enable[MONGODB-CR|SCRAM-SHA-256]  [--projectId projectId]
+func OpsManagerSecurityEnableBuilder() *cobra.Command {
+	opts := &opsManagerSecurityEnableOpts{
+		globalOpts: newGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Use:       "enable [MONGODB-CR|SCRAM-SHA-256]",
```

Comment:
[qa] why aren't we passing the authentication mechanism as an argument?

Reply:
what do you mean here exactly?

Reply:
Could you explain to me what this `enable[MONGODB-CR|SCRAM-SHA-256]` mean? Is `enable` using an argument which can be set as **MONGODB-CR** or **SCRAM-SHA-256** (I do not think so because you are not setting an argument),  or, is the command `enable` always used in the form `enable[MONGODB-CR|SCRAM-SHA-256]`?

Reply:
the values can be, MONGODB-CR, SCRAM-SHA-256 or both at the same time, check line 78

Reply:
Thank you 👍